foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
+Release: 1%{?dist}
+Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+
+Group: Applications/System
+License: GPLv2+ and GPLv3+ and BSD
+URL: https://github.com/theforeman/foreman-bootloaders
+Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+BuildArch: noarch
+
+Requires: /var/lib/tftpboot
```

Comment:
I'm on Fedora, but I changed it manually long ago - my point is that it's configurable by the proxy, so the package should read that configuration instead of extracting in the default location